scripts/pcp/update_pcp_biotype.pl

```diff
@@ -0,0 +1,142 @@
+#!/usr/bin/env perl
+
+# Copyright [1999-2015] Wellcome Trust Sanger Institute and the EMBL-European Bioinformatics Institute
+# Copyright [2016-2020] EMBL-European Bioinformatics Institute
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Updates biotype in pcp db if BOTH protein coding prediction tools agree that
+# the gene is protein coding
+
+=head1 NAME
+  update_pcp_biotype.pl
+=head1 DESCRIPTION
+Reads the output of both RNASamba and CPC2 and where they both agree that a transcript is protein coding the biotype
+is updated to pcp_protein_coding in the selected database.
+Throws if results files have non matching gene models/ number of gene models
+=head1 OPTIONS
+=head1
+=head2 Database connection options
+  -dbhost   Host name for database.
+  -dbport   What port to connect to.
+  -dbname   What name to connect to.
+  -dbuser   What username to connect as.
+  -dbpass   What password to use.
+=head2 Other options
+  --coords coordinate system name to use, default toplevel
+  --rnaSamba RNASamba output file to be used
+  --cpc2 CPC2 output file to be used
+=head1 EXAMPLES
+  ./update_pcp_biotype.pl --dbhost genebuild5 \
+    --dbname <database_name> \
+    --dbhost GBS5
+    --dport GBP5
+    --rnaSamba <RNASamba output file> \
+    --cpc2 <CPC2 output file> \
+=cut
+
+use strict;
+use warnings;
+use Getopt::Long qw(:config no_ignore_case);
+
+use Bio::EnsEMBL::Utils::Exception qw(throw);
+use Bio::EnsEMBL::DBSQL::DBAdaptor;
+
+## Some defaults
+my $coord_system = 'toplevel';
+my $dbname = '';
+my $user   = 'ensadmin';
+my $host = $ENV{GBS5};
+my $port = $ENV{GBP5};
+my $pass = 'ensembl';
```

Comment:
Can this be set to '' instead of fetching the value from an ENV? People might think they need these ENVs
$pass must be set to ''

Reply:
Done.